src/de/hasait/cipa/resource/NodeCleanup.groovy

```diff
@@ -0,0 +1,12 @@
+package de.hasait.cipa.resource
+
+import de.hasait.cipa.CipaNode
+
+/**
+ * An interface to for performing clean up of resources.
```

Comment:
Implement this interface to perform clean up of node after activities finished on node.